shared/lib/error-utils.js

```diff
@@ -126,7 +126,9 @@ function disableDesktop(backgroundConnection) {
 }
 
 export function downloadDesktopApp() {
-  global.platform.openTab({ url: 'https://metamask.io/' });
+  global.platform.openTab({
+    url: 'https://github.com/MetaMask/metamask-desktop',
```

Comment:
Can we redirect to this page:
`https://github.com/MetaMask/metamask-desktop/releases`

Reply:
Fixed